Libraries/LibWeb/HTML/Canvas/CanvasDrawImage.cpp

```diff
@@ -42,6 +42,12 @@ static void default_source_size(CanvasImageSource const& image, float& source_wi
                 source_height = source->video_height();
             }
         },
+        [&source_width, &source_height](GC::Root<OffscreenCanvas> const& source) {
```

Comment:
`LibWeb: Fix OffscreenCanvas Related errors` is very suspicious. If any of the previous commits don't build for any reason, the changes should be moved back. We want every commit to build and pass all tests in isolation.